notebooks/vertex_genai/solutions/gemini_prompt_engineering.ipynb

```diff
@@ -0,0 +1,952 @@
+{
```

Comment:
<ul><li>Maybe remove "preview" from <code>vertexai.preview.generative_models</code></li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done! Thank you! </p>